examples/Cargo.toml

```diff
@@ -0,0 +1,51 @@
+[package]
+authors = [
+    "Tide Developers",
+]
+description = "Tide web server examples"
+documentation = "https://docs.rs/tide"
+keywords = ["tide", "http", "web", "framework", "async"]
+categories = [
+  "network-programming",
+  "asynchronous",
+  "web-programming::http-server"
+]
+edition = "2018"
+license = "MIT OR Apache-2.0"
+name = "tide-examples"
+readme = "README.md"
+repository = "https://github.com/rustasync/tide"
+version = "0.1.0"
```

Comment:
Do you intend to publish these examples as a crate? If not it would be good to add `publish = false` to avoid accidental publishing (and could drop `keywords` and `categories` since they won't be used).

Reply:
Done! Thanks!